de.json

```
@@ -865,13 +865,13 @@
         "Inspector.Mesh.StatsHeader" : "Mesh Statistiken",
         "Inspector.Mesh.VertexCount" : "Vertex Anzahl: {n}",
         "Inspector.Mesh.TriangleCount" : "Dreiecks Anzahl: {n}",
-        "Inspector.Mesh.PointCount" : "Punkt Anzahl: {n}",
+        "Inspector.Mesh.PointCount" : "Punktanzahl: {n}",
         "Inspector.Mesh.SubmeshCount" : "Submesh Anzahl: {n}",
         "Inspector.Mesh.BoneCount" : "Knochen Anzahl: {n}",
         "Inspector.Mesh.BlendshapeCount" : "Blendshape Anzahl: {n}",
         "Inspector.Mesh.ChannelStats" : "Normalen: {normals}, Tangenten: {tangents}, Farben: {colors}, UV0: {uv0}, UV1: {uv1}, UV2: {uv2}, UV3: {uv3}",
 
-        "Inspector.Mesh.BakeMesh" : "Bake Mesh",
+        "Inspector.Mesh.BakeMesh" : "Fixiere Mesh",
```

"Fixiere" should be replaced as it's misleading. Literally translated it is "backen" but I couldn't find a better word but this makes more sense. 
```suggestion
        "Inspector.Mesh.BakeMesh" : "Mesh backen",
```

---

Found no better translation for it so I agree to stay with "Mesh backen"